backend/app/schemas/files.py

```
@@ -0,0 +1,47 @@
+
+
+class PresignedUploadRequest(BaseModel):
+    name: str = Field(..., description="The name of the file")
+    type: str = Field(..., description="The MIME type of the file")
+    size: int = Field(..., description="The size of the file", ge=0, le=MAX_FILE_SIZE)
+
+    @field_validator("type")
+    @classmethod
+    def validate_file_type(cls, value: str) -> str:
+        if value not in ALLOWED_MIME_TYPES:
+            raise ValueError(f"Input should be {ALLOWED_MIME_TYPES}")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)